src/fmu/tools/qcproperties/_combine_propstats.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+
```

Comment:
It is a bit hard to understand what `self` is here.
Consider either putting it in the same file as the QCProperties class or rewrite functions to not take in self ?